### ISSUE TITLE
Performance in 2.2.x

### ISSUE THREAD

**Issue by darkoverlordofdata** [photonstorm/phaser#1696]
I'm seeing a performance issue when I upgrade to the latest, 2.2.2. I've reduce it to a simple example: https://gist.github.com/darkoverlordofdata/6e47a05207a083a5923f

For performance monitoring, I'm using https://github.com/mrdoob/stats.js. The performance drops from a flat 59 FPS to a square wave shaped graph that spends most of the time in the trough at 49 FPS. It fist showed up  as jerky movement in the game.

I've regressed the issue using this example, and the issue starts when I upgrade from 2.1.3, and remains consistent up to the latest version. I've tested this in Chrome and Firefox. The issue only shows up in Chrome.

**Comment by pnstickne** [comment]
If the same slowdown observed under WebGL?

**Comment by photonstorm** [comment]
What size is your starfield.png? Also test it against 2.3-dev.

**Comment by darkoverlordofdata** [comment]
After doing more testing, this seems specific to Chrome's Linux desktop distro. I don't see the issue in Chrome for windows or android. I think this is mostly an edge case, and an issue with chrome, just annoying because I mainly use chrome on linux... But my real target market is android, so this mainly just affects my testing - I hate firefox.

I just created a test case on github to track this
https://github.com/darkoverlordofdata/phaser-test, it's visible at 
http://darkoverlordofdata.com/phaser-test/

**Comment by darkoverlordofdata** [comment]
Anyway, to answer your questions. I was using a small 320*480 image. I get the same issue with smaller images and also in webgl.

I realized this doesn't happen in the pre-upgrade code, and I'm not upgrading my phaser version, I'm upgrading from PixiJS. So assets aren't really changed. The big change was I went from using a custom tick provider to using the phaser update loop. When I revert to my custom tick, the issue goes away.

I'll continue to investigate this, but as it only affects linux desktop and I have a solution, we can go ahead and close this. If my investigation finds something relevant, I'll update you.

**Comment by darkoverlordofdata** [comment]
Ok, I got it. I implemented a plugin such that stat.begin() is called during preUpdate and stat.end() is called during postRender(). What I find then is that I get the same results across platforms, with an expected flat graph when nothing is happening in the game. So, this was probably a good strategy all along :)

I'm following the same strategy with ash, creating a plugin, and it seems to run most smoothly when ticked in the post render phase.
